pallets/handles/README.md

```diff
@@ -0,0 +1,74 @@
+| `retire_handle`<br />Retiren a handle. Retired handles + suffix are never reused.   | MSA Owner | Free  | [`HandleRetired`](https://frequency-chain.github.io/frequency/pallet_handles/pallet/enum.Event.html#variant.HandleRetired) | 27             |
+| `change_handle`<br />Convenous method to retire and then claim a new handle  | Provider or MSA Owner | Capacity or Tokens  | [`HandleRetired`](https://frequency-chain.github.io/frequency/pallet_handles/pallet/enum.Event.html#variant.HandleRetired), [`HandleClaimed`](https://frequency-chain.github.io/frequency/{pallet_name}/pallet/enum.Event.html#variant.HandleClaimed) | 47             |
+
+See [Rust Docs](https://frequency-chain.github.io/frequency/pallet_handles/pallet/struct.Pallet.html) for more details.
+
+### State Queries
+
+Note: RPC use is suggested over the direct state queries for handles.
+
+| Name      | Description         | Query                    | Runtime Added |
+| --------- | ------------------- | ------------------------ | ------------- |
+| Get Handle by MSA Id  | Returns the Display Handle and the block it was claimed   | `msaIdToDisplayName` | 29             |
+| Get MSA Id by Canonical and Suffix  | Uses the stored canonical lookup string NOT the display handle with the suffix to retrieve the MSA Id   | `canonicalBaseHandleAndSuffixToMSAId` | 29             |
+
+See the [Rust Docs](https://frequency-chain.github.io/frequency/pallet_handles/pallet/storage_types/index.html) for additional state queries and details.
+
+### RPCs
+
+Note: May be restricted based on node settings and configuration.
+
+| Name    | Description       | Call                                                                                                 | Node Version |
+| ------- | ----------------- | ---------------------------------------------------------------------------------------------------- | ------------ |
+| Get Handle by MSA Id | Returns the base handle and suffix as well as the cononical version of the handle | [`getHandleforMSA`](https://frequency-chain.github.io/frequency/pallet_handles_rpc/trait.HandlesApiServer.html#tymethod.get_handle_for_msa) | v1.6.0+      |
+| Get MSA Id by Display Handle | Returns the MSA Id for a given Display Handle | [`getMsaForHandle`](https://frequency-chain.github.io/frequency/pallet_handles_rpc/trait.HandlesApiServer.html#tymethod.get_handle_for_msa) | v1.6.0+      |
+| Validate Handle String | Checks to see if the handle string validates | [`validateHandle`](https://frequency-chain.github.io/frequency/pallet_handles_rpc/trait.HandlesApiServer.html#tymethod.validate_handle) | v1.8.0+      |
+| Get Next Suffixes | Given a Base Handle and count, returns the next suffixes that will be used for claimed handles | [`getNextSuffixes`](https://frequency-chain.github.io/frequency/pallet_handles_rpc/trait.HandlesApiServer.html#tymethod.get_next_suffixes) | v1.8.0+      |
```

Comment:
To this point--should we say anything in the docs about the fact that the shuffled suffixes are deterministic for each base handle, not random?